altsrc/flag_generated.go

```diff
@@ -141,6 +183,13 @@ func (f *IntSliceFlag) Apply(set *flag.FlagSet) {
 	f.IntSliceFlag.Apply(set)
 }
 
+// ApplyWithError saves the flagSet for later usage calls, then calls the
+// wrapped IntSliceFlag.ApplyWithError
+func (f *IntSliceFlag) ApplyWithError(set *flag.FlagSet) error {
```

Comment:
I was thinking in `v2` we could just switch the signature of `Apply` to return `error`, but I'm OK with doing that as a second pass since this is just an additive change as-is.